clients/go/test/api_uploads_test.go

```diff
@@ -0,0 +1,77 @@
+/*
+Phrase Strings API Reference
+
+Testing UploadsApiService
+
+*/
+
+// Code generated by OpenAPI Generator (https://openapi-generator.tech);
+
+package phrase
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/phrase/phrase-go"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_phrase_UploadsApiService(t *testing.T) {
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		// Send the mock response
+		response := `{"foo": "bar"}`
+		w.Header().Set("Content-Type", "application/json")
+		w.WriteHeader(http.StatusOK)
+
+		w.Write([]byte(response))
+	}))
+
+	defer server.Close()
+
+	configuration := phrase.NewConfiguration()
+	configuration.BasePath = server.URL
+	apiClient := phrase.NewAPIClient(configuration)
+
+	t.Run("Test UploadsApiService UploadCreate", func(t *testing.T) {
+		localVarOptionals := phrase.UploadCreateOpts{}
```

Comment:
not sure how easy would it be to pass an actual file to this call and then assert that the body contains it?

Reply:
Looked into it, tried to access request via 

```
httpRes.Request.Body which but it returns {}
```

Also tried to explore possibility to read the request Body inside the [mock server](https://github.com/phrase/openapi/pull/440/files#diff-8208c8a7de5d4c0b513c42d411c1f576a37ec316caa3d0dc4dccbed902cc150dR27) but I will have to look further how to handle the returned Body object of type `io.ReadCloser` which by default is unreadable. If its important, I can further investigate if its possible.